### DISCUSSION TITLE
Soft Hardcore mode

### DISCUSSION THREAD

**Discussion by duarm** [FakeFishGames/Barotrauma#11580] [Ideas and suggestions]
Basically, a mode in which once you die, you create a new character, you  may opt-in losing your talents/skills, but by default, you only create a new character with a different name and appearance. This would severely improve the atmosphere of the game without any mechanical punishment. Maybe a board showing all of our comrades who didn't make it through the journey, like the one it shows every mission end, but for the whole campaign.

**Comment by Ek-Videogames** [comment]
Neat idea - so like the "fixed" respawn mechanics back from v0.13.0.11 with character randomizer added onto it.
